homework/jaellysbales/Horoscope/app/src/main/java/com/jaellysbales/horoscope/drawer/NavDrawerItem.java

```diff
@@ -0,0 +1,35 @@
+package com.jaellysbales.horoscope.drawer;
+
+/**
+ * Created by jaellysbales on 5/8/15.
+ */
+public class NavDrawerItem {
+    private boolean showNotify;
+    private String title;
+
+
+    public NavDrawerItem() {
+
+    }
+
+    public NavDrawerItem(boolean showNotify, String title) {
+        this.showNotify = showNotify;
+        this.title = title;
+    }
+
+    public boolean isShowNotify() {
```

Comment:
use meaningful variable names. isShowNotify is not that intuitive though it sticks to boolean naming conventions (isTrue, isFalse, etc). what does show notify do?